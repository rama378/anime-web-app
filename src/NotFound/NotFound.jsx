import './NotFound.css';

const NotFound = () => {
  return (
    <>404</>
  );
}

export default NotFound;
