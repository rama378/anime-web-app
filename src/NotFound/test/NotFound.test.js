import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import NotFound from '../NotFound';

const renderer = new ShallowRenderer();

describe('<NotFound />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<NotFound />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
