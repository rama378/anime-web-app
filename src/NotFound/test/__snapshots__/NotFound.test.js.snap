// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render and match the snapshot 1`] = `
<React.Fragment>
  404
</React.Fragment>
`;
